DOCS/man/input.rst

```diff
@@ -663,6 +663,12 @@ Track Manipulation
         (In this case, title/language are ignored, and if the was changed since
         it was loaded, these changes won't be reflected.)
 
+    Additionally the following flags can be added with a ``+``:
+
+    <hearing-impaired>
```

Comment:
Just a thought but how do you feel about combining this with `visual-impaired`. i.e. if it's an audio track mark it as a visually impaired? I guess a video track would be a no-op with this flag.

Reply:
Sure, can do. Generally we could add all the track flags, but might be overkill, I think most of them are not that useful. We can do on case by case requests.

Reply:
done